modules/custom/openy_migrate/src/Importer.php

```diff
@@ -69,4 +80,26 @@ public function importByTag($migration_tag) {
     }
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function destroyMigrationData($module_name) {
+    // Get config entities that are dependent on module.
+    $dependencies = $this->configManager->findConfigEntityDependentsAsEntities('module', (array) $module_name);
```

Comment:
$dependencies variable can be empty. Let's add a condition to make sure you have something there